installer/tue-install-impl.bash

```diff
@@ -670,14 +691,21 @@ function tue-install-ppa-now
 {
     tue-install-debug "tue-install-ppa-now $*"
 
+    if [ -z "$1" ]
+    then
+        tue-install-error "Invalid tue-install-ppa-now call: needs ppa as argument."
+    fi
+
     local PPA_ADDED=""
     # shellcheck disable=SC2048
     for ppa in $*
     do
-        if ! grep -h "^deb.*${ppa#ppa:}" /etc/apt/sources.list.d/* 1>/dev/null 2>&1
+        ppa="${ppa//^/ }"
+        if ! grep -q -h "^deb.*${ppa#ppa:}" /etc/apt/sources.list.d/* 2>&1
```

Comment:
What about the `deb` source ppa? Are you sure the variable substitution will work for ppas with `deb <src>`?

Reply:
The debs will be added tot /etc/apt/sources.list. So this will not work